docs/auth/README.adoc

```diff
@@ -0,0 +1,123 @@
+= AeroGear Services Auth SDK
+
+Mobile authentication SDK based on link:http://www.keycloak.org/[Keycloak] using link:http://openid.net/connect/[OpenID Connect].
+
+== Usage
+
+To use the Auth SDK you'll first need to:
+
+* Have a Keycloak instance. See the <<Setting up Keycloak>> section.
+* Import the Core module. See link:./core/README.adoc[the Core documentation].
```

Comment:
Do we still need the core module imported?

Reply:
No. Core README will be gone in @paolobueno PR.